index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- stylesheet -->
+		<link rel="stylesheet" href="style.css" />
+		<!-- fontawesome -->
+			<script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+		<!-- google font -->
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=M+PLUS+Rounded+1c:wght@100;300;400;500;700&family=Poppins:wght@300;400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+		<title>Portfolio Website</title>
+	</head>
+	<body>
+		<div class="intro-container">
+			<header>
+				<h1>Victor</h1>
+				<div class="hamburger">
+					<i class="fa fa-bars fa-1x"></i>
+				</div>
```

Comment:
- [ ]  Please consider wrapping the hamburger menu with a `nav` tag instead of a `div` tag as the hamburger menu consists of navigation links and also to improve the semantic structure of your HTML😄. You can read more on[ SEMANTIC HTML.](https://www.w3schools.com/html/html5_semantic_elements.asp).

 See my suggestions below.

```suggestion
				<nav class="hamburger">
					<i class="fa fa-bars fa-1x"></i>
				</nav>
```

Reply:
This is a piece of great information. Thank you.